From fc1d468cca56873b54af58fb12d4cb962fe96a77 Mon Sep 17 00:00:00 2001
From: Vaibhav Bedia <vaibhav.bedia@ti.com>
Date: Fri, 9 Mar 2012 21:17:48 +0530
Subject: [PATCH 0881/1092] ARM: OMAP: AM33XX: Rename the firmware binary

Update the name of the firmware binary to be loaded
onto Cortex-M3 for low power transitions.

If the binary is not provided at the time of kernel
compilation there's a timeout of 60 seconds. To avoid
creating any confusion due to missing binary add a
message indicating that the firmware is now going
to be loaded.

Signed-off-by: Vaibhav Bedia <vaibhav.bedia@ti.com>
---
 arch/arm/mach-omap2/pm33xx.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/arch/arm/mach-omap2/pm33xx.c b/arch/arm/mach-omap2/pm33xx.c
index bdb0864..23e122d 100644
--- a/arch/arm/mach-omap2/pm33xx.c
+++ b/arch/arm/mach-omap2/pm33xx.c
@@ -371,8 +371,10 @@ static int wkup_m3_init(void)
 		goto err5;
 	}
 
+	pr_info("Trying to load am335x-pm-firmware.bin\n");
+
 	/* Now try to load the firware */
-	ret = request_firmware(&firmware, "cm3-firmware.bin", mpu_dev);
+	ret = request_firmware(&firmware, "am335x-pm-firmware.bin", mpu_dev);
 	if (ret < 0) {
 		dev_err(mpu_dev, "request_firmware failed\n");
 		goto err6;
-- 
1.7.11.2

